testprovider/requirements.txt

```diff
@@ -1,4 +1,4 @@
-django<3
+django==2.2.12
```

Comment:
This means we have to update periodically to keep up with Django 2.x releases, but this is test infrastructure so I think that's ok. I'll keep up with it.

If it's the case that this doesn't alleviate the GitHub security vulnerability notifications, then I'll back it out.